packages/amplify-graphql-sql-transformer/src/graphql-sql-transformer.ts

```diff
@@ -105,13 +105,32 @@ export class SqlTransformer extends TransformerPluginBase {
           ),
         );
 
+        resolver.addToSlot(
+          'postAuth',
+          MappingTemplate.s3MappingTemplateFromString(
+            generateAuthExpressionForSandboxMode(context.transformParameters.sandboxModeEnabled),
+            `${config.resolverTypeName}.${config.resolverFieldName}.{slotName}.{slotIndex}.req.vtl`,
+          ),
+        );
```

Comment:
Having atleast a unit test for these resolver changes would help.

Reply:
Other alternative is to pull these changes into another sub-feature branch in the repo and merge post all E2Es pass.